puppet/modules/freight/templates/freight.conf.erb

```diff
@@ -17,5 +17,5 @@ GPG="563278F6"
 # Follow symlinks in VARLIB
 SYMLINKS="on"
 
-# x86 plus supported ARM platforms
-ARCHS="i386 amd64 armhf arm64"
+# x86
+ARCHS="i386 amd64"
```

Comment:
I debated this, not sure if this is the right one

Reply:
I think we can also remove i386, iirc that was for 32 bit builds

Reply:
Not sure, but looking at https://deb.theforeman.org/pool/buster/nightly/r/ruby-journald-native/ we still build these. I'm a bit unsure about removing them. I also don't know if this removes the arm builds from all the existing repos.

@evgeni any idea?

Reply:
I would expect it does, so I'd leave it here.